components/Community.jsx

```diff
@@ -18,7 +18,7 @@ const Community = () => {
         </p>
       </div>
       <h1 className='px-5 text-3xl md:mt-10'>
-        Join a Large Growing Community in Town
+        Join a Large Open Source Growing Community in Town
```

Comment:
@Ronniedeveloper how if we had this wording as 
```suggestion
        Join a Large Growing Open Source Community in Town
```

Reply:
I like this suggestion 💯